zkevm-circuits/Cargo.toml

```
@@ -57,7 +57,7 @@ paste = "1.0"
 default = ["test", "test-circuits", "debug-annotations", "parallel_syn"]
 test = ["mock", "bus-mapping/test"]
 
-scroll = ["bus-mapping/scroll", "eth-types/scroll", "mock?/scroll", "zktrie", "poseidon-codehash"]
+scroll = ["bus-mapping/scroll", "eth-types/scroll", "mock?/scroll", "zktrie", "poseidon-codehash", "dual_bytecode"]
```

```suggestion
scroll = ["bus-mapping/scroll", "eth-types/scroll", "mock?/scroll", "zktrie", "poseidon-codehash", "dual-bytecode"]
```

We should use `-` over `_` for crate names and feature names.

---

good finding, will updated it.

---

renamed in [0786aca](https://github.com/scroll-tech/zkevm-circuits/pull/1368/commits/0786aca112368865e8c4eccdd657a213cf272b9e)